rebar.config

```diff
@@ -14,13 +12,22 @@
             warn_unused_import,
             warnings_as_errors]}.
```

Comment:
Yes I want debug_info in the `erl_opts` list because it makes using tools like redbug and so forth much easier.